Event Grid Explorer - Adding Support for Event Grid V2 (#737)
paolosalvatori/ServiceBusExplorer: EventGridExplorer_README.md

This additional support for Event Grid enables users to test event delivery with ease, contributing to the adoption of the new service.

## Connect to Event Grid

Under the File tab, the user can connect to an Event Grid V2 namespace by providing the Resource Group, Namespace Name, Subscription ID, and API Version found in the Azure Portal.

![Connect to EGV2](./media/connect-event-grid.png)

![Connection Info](./media/connect-info-event-grid.png)

## Main View

When the connection is successful, a tree view of the Event Grid V2 namespace with its contained topics and subscriptions is displayed along with properties views.
Right-clicking the Topics and Subscriptions nodes enables the user to create topics/subscriptions and right-clicking the existing topics and subscriptions enables the user to publish/receive events and delete these entities.

![Main View](./media/main-view-event-grid.png)

## Publish

When right-clicking the Publish option on a topic, the user can create and publish an event by entering the event source, type and JSON payload. 

![Publish Event to Topic](./media/publish-event-grid.png)

## Receive

> When receiving events through a subscription, the user can receive the maximum number of events (100) or a top number of events.

Reviewer: Is it "up to 100 messages from the top of the subscription"?

When successful, the data view table is populated with the received events and the event data is displayed in a JSON format.

![Receive Event Popup](./media/receive-popup-event-grid.png)

![Receive Events in Subscription](./media/receive-results-event-grid.png)

> ## Acknowledge/Release/Reject

Reviewer: ```suggestion
## Event settlement
```

Reviewer: @t-wangamy, are these operations not settlement?

Author: Made this change but somehow it was undone, I'll update it

> On the received events listed in the table, the user can multi-select events to perform the acknowledge/release/reject operations.

Reviewer: ```suggestion
The user can multi-select the received events in the table to acknowledge, release, or reject events.
```
